jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -722,6 +811,43 @@ ecma_date_timezone_offset (ecma_number_t time) /**< time value */
 } /* ecma_date_timezone_offset */
 
 /**
+ * Helper function to set Date internal property.
+ *
+ * Used by:
+ *         - All Date.prototype.set* routine except Date.prototype.setTime.
+ *
+ * @return completion value
```

Comment:
@szledan, usually, completion values are returned by specification-defined routines. In the cases, content of completion value is clear, while content of this function's completion is not - without looking into its code.
Could you, please, extend the comment with note about what value is returned from the function, or with a reference to specification, so that origin of the value becomes clear?